### ISSUE TITLE
Stealing from outposts is way too easy/consistent/profitable

### ISSUE THREAD

**Issue by ubertpendragon** [Regalis11/Barotrauma#10621]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Most players consistently steal from outpsts, with very little difficulty. 

Some rooms are completely out of LOS, which means NPCs will never see you stealing

You can at least steal a diving suit and 4x oxygen tanks every single outpost, that's AT LEAST 1 Titalloy, 1 Rubber and 4 Aluminum each visit. Sometimes multiple modules have diving suits

![image](https://user-images.githubusercontent.com/73229309/204803554-b07b84bb-c8ca-4cdd-a677-66be3648849b.png)
![image](https://user-images.githubusercontent.com/73229309/204803714-408da787-524b-41fe-9ed1-54b76b84516a.png)
![image](https://user-images.githubusercontent.com/73229309/204805116-24d14006-0f4d-4edb-b6c6-8211c43d5c01.png)

There other modules suffering from the same problem:
![image](https://user-images.githubusercontent.com/73229309/204804789-6aa55d26-cee1-4abe-8b22-48c27a711846.png)

Players have also gotten into the habit of using Fireman's carry to drag NPCs out of rooms 
![image](https://user-images.githubusercontent.com/73229309/204804910-f9884565-610d-44b6-a613-43e0df64896e.png)

You could also distract the entire security personel by triggering their AI on the fact corner of the station for -1 rep and then loot everything else (especially useful in colonies)

In the unstable/factions build you can now find high quality loot in cabinets
![image](https://user-images.githubusercontent.com/73229309/204806099-3d2f682a-1f6e-435f-97d1-b6af1218755e.png)

------

In short pillaging outposts needs a big nerf:

- **Lose reputation whether you get seen or not.**  That's the big one, but also the only consistent way to not make stealing a consequence-free resource printing machine
- **Make it impossible to damage station walls.** All of these repair supplies only exist because colony walls can be broken, which sends the AI into chaos and has a massive impact on performance in MP. Also because OUTPOSTS ARE NOT BUILT TO BE DRAINED  
- Replace valuable items like Diving Suits with Disposable suits
- Change room layouts so it isn't as easy
- Reduce the amount of loot (colonies don't need every supply cabinet loaded with oxygen tanks, diving masks and high quality welding tools)

**Comment by mygamingaccount** [comment]
If it wasn't for stealing, a lot of us wouldn't go for shore leave

**Comment by BiffW** [comment]
Don't go against players too extremely. Let's say we can overhaul this behaviour by linking it to the reputation system.
For example:
When your rep is low, guards will follow players in the station to prevent them from stealing.
When your rep is high, npcs will consider player a member of the family and you can take anything as you wish without punishment, so players are not stealing literally.

**Comment by mygamingaccount** [comment]
> guards will follow players in the station to prevent them from stealing

OMG that is brilliant!!!

**Comment by JeroonBosch** [comment]
CrewModule_01
CrewModule_02
EngineeringModule_01
seem to be the worst offenders here on regular outposts

**Comment by ubertpendragon** [comment]
From our latest session, this is a single residential module
![image](https://user-images.githubusercontent.com/73229309/212495764-5d12bf31-72b1-415f-a46c-e915972cb831.png)

**Comment by Regalis11** [comment]
Created an implementation ticket in our public repo https://github.com/Regalis11/Barotrauma-development/issues/3905, closing this one
